ert3/console/_console.py

```diff
@@ -126,6 +127,10 @@ def _clean(workspace, args):
 def main():
     try:
         _main()
+    except ert3.exceptions.ConfigValidationError as e:
+        ert3.console.report_validation_errors(str(e), source=e.source)
+    except ValidationError as e:
```

Comment:
The ValidationError should not bubble up, we should catch it early, and turn it into the ConfigValidationError, so that the source can be added. I left the ValidationError in with the thought that if we ever use pydantic somewhere but do not use the new error, then at least we get pretty printing.

But now you mention it, I am thinking that might be a bad idea. If somebody uses pydantic, and does not turn ValidationError in ConfigValidationError, that should be a (minor) bug. Then we will se a raw exception trace, which should be a hint to do something about it.... 

So maybe I should remove that, what do you think?

Reply:
I agree. If non-ERT specific exceptions reaches all the way to the cli, it is a hard failure and we should just display the stack trace 👍🏻